<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-09-07T22:42:58 EEST" name="Suite" finished-at="2023-09-07T22:43:20 EEST" duration-ms="22299">
    <groups>
    </groups>
    <test started-at="2023-09-07T22:42:58 EEST" name="Test" finished-at="2023-09-07T22:43:16 EEST" duration-ms="17593">
      <class name="uk.co.automationtesting.AddRemoveItemBasketTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:uk.co.automationtesting.AddRemoveItemBasketTest@223aa2f7]" started-at="2023-09-07T22:42:59 EEST" name="setup" finished-at="2023-09-07T22:43:04 EEST" duration-ms="5345" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="addRemoveItem()[pri:0, instance:uk.co.automationtesting.AddRemoveItemBasketTest@223aa2f7]" started-at="2023-09-07T22:43:04 EEST" name="addRemoveItem" finished-at="2023-09-07T22:43:14 EEST" duration-ms="10579" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Cart amount did not match the expected amount, it foud$45.24expected $45.23]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Cart amount did not match the expected amount, it foud$45.24expected $45.23
at org.testng.Assert.fail(Assert.java:99)
at uk.co.automationtesting.AddRemoveItemBasketTest.addRemoveItem(AddRemoveItemBasketTest.java:75)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addRemoveItem -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:uk.co.automationtesting.AddRemoveItemBasketTest@223aa2f7]" started-at="2023-09-07T22:43:15 EEST" name="tearDown" finished-at="2023-09-07T22:43:16 EEST" duration-ms="984" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- uk.co.automationtesting.AddRemoveItemBasketTest -->
    </test> <!-- Test -->
    <test started-at="2023-09-07T22:42:58 EEST" name="Test2" finished-at="2023-09-07T22:43:20 EEST" duration-ms="22299">
      <class name="uk.co.automationtesting.OrderCompleteTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:uk.co.automationtesting.OrderCompleteTest@f736069]" started-at="2023-09-07T22:42:59 EEST" name="setup" finished-at="2023-09-07T22:43:04 EEST" duration-ms="5536" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="endToEndTest()[pri:0, instance:uk.co.automationtesting.OrderCompleteTest@f736069]" started-at="2023-09-07T22:43:04 EEST" name="endToEndTest" finished-at="2023-09-07T22:43:19 EEST" duration-ms="15217" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endToEndTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:uk.co.automationtesting.OrderCompleteTest@f736069]" started-at="2023-09-07T22:43:19 EEST" name="tearDown" finished-at="2023-09-07T22:43:20 EEST" duration-ms="1192" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- uk.co.automationtesting.OrderCompleteTest -->
    </test> <!-- Test2 -->
  </suite> <!-- Suite -->
</testng-results>
